webapp/src/pages/UserPage/UserPage.css

```diff
@@ -0,0 +1,84 @@
+.user-page {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  grid-column-gap: var(--column-gap);
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 1fr auto;
+  grid-row-gap: 4rem;
+}
+
+.user-wrapper {
+  height: 24.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.user-wrapper .user-text {
+  height: 3.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.user-wrapper .user-text > * {
+  margin: 0;
+}
+
+.user-page .button {
+  justify-self: center;
+}
+
+@media all and (min-width: 768px) {
+  .user-page {
+    display: grid;
+    grid-template-columns: repeat(var(--column-number), 1fr);
+    grid-template-rows: auto 1fr auto;
+    grid-row-gap: 8rem; /* this size doesn't look like in figma */
+  }
+  .user-page .header {
+    grid-column: 1 / var(--column-end);
+  }
+
+  .user-wrapper {
+    height: 10.2rem;
+    grid-column: 3 / calc(var(--column-end) - 1);
+    grid-row: 2/3;
+    display: grid;
```

Comment:
He hecho una grid dentro de una celda de otra grid... a lo mejor no es la mejor idea, pero lo podemos discutir.

Reply:
I think this is a good approach :+1:. There is nothing wrong with having a grid/flexbox container inside another grid/flexbox item.